### DISCUSSION TITLE
[Help]: Disable any keyboard event for sliding

### DISCUSSION THREAD

**Discussion by harshpandey002** [davidjerleke/embla-carousel#854] [Help] [answered]
### Summary

I'm creating a TypeForm-like experience for my client. My carousel is vertical and covers the whole page. There are single/multiple inputs on each slide.

When I'm not focused on inputs, Left/Right arrows doesn't triggers the sliding (which is cool), but when I'm focussed on any input, those arrow keys makes the carousel slide and does not allow me to navigate inside the input.

Is there any way I can make it such that the carousel only slides from the code i.e. when `api.scrollNext()` or others are triggered?

I may want to keep `Escape` key to trigger `api.scrollPrev()`, but I'm okay if I have to loose that.

Also, is there any way I can disable focus being shifted to the next input when I press the `Tab` key?

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
@harshpandey002 thanks for your question. I believe this is a duplicate of:

- #822

**Reply by harshpandey002** [reply]
Oh my bad, thanks @davidjerleke

**Reply by davidjerleke** [reply]
@harshpandey002 this feature has been included in the [v8.2.0 release](https://github.com/davidjerleke/embla-carousel/releases/tag/v8.2.0).
